apps/ledger-live-desktop/static/i18n/en/app.json

```diff
@@ -2027,7 +2030,7 @@
         "energyDescription": "Energy points are required to execute smart contracts. If you don't run any smart contracts, there is no need to choose rewards in Energy.",
         "amountLabel": "Amount to freeze",
         "available": "Available: {{amountAvailable}}",
-        "info": "Frozen assets cannot be sent for 3 days."
+        "info": "Frozen assets can be unfrozen immediately and accessible after the unfreezing period."
```

Comment:
How long is the unfreezing period with Tron staking v2?

Reply:
14 days. Will add it to this phrase.

Reply:
@Philippoes You can add it on the mobile translations as well. Thanks.